src/e2e/step-definitions/open-forms.ts

```diff
@@ -25,28 +25,33 @@ Given(
     const id = uniqid();
     this.testStorage.set("open-forms-testid", id);
 
+    const firstNameInput = this.page.getByLabel("Voornaam").nth(0);
```

Comment:
Don't we need a regex option to select by the Dutch or English version?

TBH - can't we set the e2e browser locale or language to Dutch? We are not that interested in the English version.